### ISSUE TITLE
[BUG] `Exception ignored deletion of interned string failed`

### ISSUE THREAD

**Issue by AlecHaring** [dashingsoft/pyarmor#1068]
I'm having trouble figuring out the exact cause, or what file in my project is causing the problem, but I'm getting the following error when executing the following command:
```
pyarmor -d gen --enable-rft .\fake_name\
```

```
...
INFO     fake_name.util.updater.update_modal:1:<module>
INFO     fake_name.util.updater.update_modal:7:pyarmor__4108
DEBUG    process inline marker
INFO     fake_name.util.updater.__init__:1:<module>
INFO     write dist\fake_name\util\updater\__init__.py
INFO     obfuscate scripts OK
Exception ignored deletion of interned string failed:
KeyError: 'ý55ªÒ4 èPEY\tk×7ä'
Exception ignored deletion of interned string failed:
KeyError: 'ý55ªÒ4 èPER\x1emÏ*åËû5\x99}À\x8bù\x87'
Exception ignored deletion of interned string failed:
KeyError: 'þ=)»Û#-èO\x12'
```

Any suggestions on how to debug? It doesn't output a `pyarmor.error.log`

**Comment by jondy** [comment]
I'll check the related source first, it may be caused by object refcount.

**Comment by jondy** [comment]
It may be caused by same reason as #1070, the encrypted string overwrites the original string.

It will be fixed in next release.

**Comment by jondy** [comment]
Fixed in dev version `8.1.dev2`
